filebeat/module/elasticsearch/audit/ingest/pipeline-json.json

```diff
@@ -66,28 +74,28 @@
         },
         {
             "set": {
-                "if": "ctx.elasticsearch.audit?.url?.path != null && ctx.elasticsearch.audit?.url?.query == null",
                 "field": "url.original",
+                "if": "ctx.elasticsearch.audit?.url?.path != null \u0026\u0026 ctx.elasticsearch.audit?.url?.query == null",
```

Comment:
It's not nice that it changes the sorting as in some cases the sorting might have been for better readability.

Reply:
@exekias @jsoriano Any thoughts around the change to the sorting order? Problem or not?

Reply:
I'd prefer to keep the order given by the developer, for example I think it is good to have conditions in the first place, so they don't get hidden between other options, also we use to place the `on_failure` blocks at the end, and they are now before the processors, this can be confusing.